codeforamerica/vita-min
# Nj 165 dep health ins

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/165

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- Add dependent checkboxes to existing health insurance eligibility screen
- Add dependent health insurance status to nj 1040 and auxiliary dependent pdf
- Make email and phone autocomplete (just because it was driving me nuts, but lmk if I should walk that back)

Note that this will be updated once we settle the question of what to do for the case where self/spouse didn't have coverage, so there are a few todo notes and I didn't go for pixel-perfect on the design.

## How to test?
- Choose the many deps claimed as dep profile
- Check "no" for health coverage, check a few dependents
- See PDF and XML indications of lack of healthcare coverage for those dependents

## Screenshots (for visual changes)
- Before
![image](https://github.com/user-attachments/assets/7736e809-f8ff-4c32-8022-07daca605dad)

- After
![image](https://github.com/user-attachments/assets/1015bbcf-63ac-4d38-90f9-559b2256a8c9)
![Screenshot 2024-12-12 at 12-27-42 submission pdf](https://github.com/user-attachments/assets/abca4096-9ed2-472d-ae8d-5ac65e35c1c2)
![Screenshot 2024-12-12 at 12-27-55 submission pdf](https://github.com/user-attachments/assets/74d511d5-9af7-43cc-a2ed-4ac9bcedb2b7)
